src/components/AntiquityCard/AntiquityCard.story.tsx

```diff
@@ -0,0 +1,11 @@
+import { AntiquityCard } from './AntiquityCard';
+
+export default {
+  title: 'Components/Antiquities',
+  component: AntiquityCard,
+  tags: ['autodocs'],
+};
+
+export function Example() {
+  return <AntiquityCard />;
+}
```

Comment:
```suggestion
export const Default = {};
```
Wouldn't this be enough?